plugins/woocommerce/src/Internal/ProductAttributesLookup/LookupDataStore.php

```diff
@@ -136,29 +128,6 @@ public function check_lookup_table_exists() {
 		return $this->lookup_table_name === $wpdb->get_var( $query );
 	}
 
-	/**
-	 * Checks if the feature is visible (so that dedicated entries will be added to the debug tools page).
-	 *
-	 * @return bool True if the feature is visible.
-	 */
-	public function is_feature_visible() {
-		return $this->is_feature_visible;
-	}
-
-	/**
-	 * Makes the feature visible, so that dedicated entries will be added to the debug tools page.
-	 */
-	public function show_feature() {
-		$this->is_feature_visible = true;
-	}
-
-	/**
-	 * Hides the feature, so that no entries will be added to the debug tools page.
-	 */
-	public function hide_feature() {
-		$this->is_feature_visible = false;
-	}
-
```

Comment:
We included a README in the `Internal` directory with a loud warning about the code in that directory not having any guarantee of consistency between WooCommerce versions (which is the very purpose of that directory). I don't think we did any other specific communication about this, but if we eventually do, I think it would be part of a broader set of guidelines for adding new code to WooCommerce.